tools/signing/sign.targets

```diff
@@ -0,0 +1,61 @@
+<Project DefaultTargets="Sign">
+  <Import Project="..\dependencies.props" />
+  <PropertyGroup>
+    <NuGetPackageRoot>$(RestorePackagesPath)</NuGetPackageRoot>
+    <NuGetPackageRoot Condition="!HasTrailingSlash('$(NuGetPackageRoot)')">$(NuGetPackageRoot)\</NuGetPackageRoot>
+    <SignType Condition="'$(SignType)' == ''">test</SignType> 
+
+    <_DryRun>true</_DryRun>
+    <_DryRun Condition="'$(BUILD_BUILDNUMBER)' != ''">false</_DryRun>
+    <_TestSign>false</_TestSign>
+    <_TestSign Condition="'$(SignType)' == 'test'">true</_TestSign>
+
+    <_DesktopMSBuildRequired Condition="'$(_DryRun)' != 'true'">true</_DesktopMSBuildRequired>
+
+    <_SigningTempDir Condition="'$(_SigningTempDir)' == ''">$(MSBuildProjectDirectory)\obj\</_SigningTempDir>
```

Comment:
You may want to add a `tmp` directory here. A bunch of folder\files will be created during the signing.